aws_db_instance.tf

```diff
@@ -3,47 +3,53 @@
 #}
 
 resource "aws_db_instance" "secureInstance" {
-  allocated_storage    = 20
-  storage_type         = "gp2"
-  engine               = "mysql"
-  engine_version       = "5.7"
-  instance_class       = "db.t2.micro"
-  name                 = "mydb"
-  backup_retention_period = 90
+  allocated_storage                   = 20
+  storage_type                        = "gp2"
+  engine                              = "mysql"
+  engine_version                      = "5.7"
+  instance_class                      = "db.m5.xlarge"
+  name                                = "mydb"
+  backup_retention_period             = 90
   iam_database_authentication_enabled = true
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: RDS Database Master Username**
 
**Vulnerability**
Ensure AWS RDS instances are using secure and unique master usernames for their databases.
 
**Remediation**
Ensure that username is not set to 'awsuser'

source: accurics-